app/views/application_mailer/_header.html.erb

```diff
@@ -0,0 +1,7 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
```

Comment:
My suggestion was to use layouts rather than partials for these - is that possible?  At least you'd not have two parts of the same piece of HTML split across files.